app/src/main/java/me/ccrama/redditslide/Activities/Settings.java

```diff
@@ -132,55 +304,73 @@ public void onClick(DialogInterface dialog,
             });
         }
 
-        findViewById(R.id.general).setOnClickListener(new OnSingleClickListener() {
+        ((EditText) findViewById(R.id.settings_search)).addTextChangedListener(new TextWatcher() {
+
+            @Override
+            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) { }
+
```

Comment:
I'm sure there is a real reason but listeners in Android are hell states because the lifecycle can just call these for haphazard reasons.

Reply:
Honestly this might have just been from a code template, unless it disqualifies the class as the listener I think we should remove it

Reply:
TextWatcher is an interface, so I believe all of the methods have to be implemented -- even if they don't do anything.

Reply:
Awesome, what I thought